modules/ROOT/pages/worker-monitoring.adoc

```diff
@@ -32,7 +32,7 @@ If you are setting this from the *Deploy Application* menu, click *Deploy Applic
 
 If you are setting this from the *Manage Application* menu, click *Apply Changes*.
 
-Upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
+For applications running on runtime 3.x.y, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```

Comment:
For applications running on Mule 3.x.x...

Reply:
```suggestion
For applications running on runtime 3.8.x and below, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```

Reply:
Don't use "below." 3.8.x and earlier (less ambiguous)